frontend/src/visualization/filtering/Filter.tsx

```diff
@@ -66,139 +71,87 @@ function TabPanel(props: TabPanelProps) {
 }
 
 /**
- * Fetches nodeTypes from the schemaService.
- *
- * @param schemaService - the schemaService the data is fetched from
- * @param cancellation - the cancellation token
- */
-function fetchNodeTypes(
-  schemaService: SchemaService,
-  cancellation: CancellationToken
-): Promise<NodeType[]> {
-  return schemaService.getNodeTypes(cancellation);
-}
-
-/**
- * Fetches edgeTypes from the schemaService.
- *
- * @param schemaService - the schemaService the data is fetched from
- * @param cancellation - the cancellation token
+ * The filtering sidebar component used to filter specific elements of the {@link Graph}.
```

Comment:
The goal of this refactoring step was to reduce/remove the coupling of Filter<->Graph. It's true that atm the filter is used only here, but there might be usages elsewhere. Just remove the graph part as it implies coupling that does not exists directly between these classes.
This component does not to update the Graph, but the FilterStateStore.
```suggestion
 * The filtering sidebar component used to update the {@link FilterStateStore}.
```